pkg/cmd/pipeline/logs.go

```diff
@@ -53,15 +53,25 @@ func logCommand(p cli.Params) *cobra.Command {
 	}
 
 	eg := `
-  # show the logs of the latest pipelinerun of given pipeline
-	tkn pipeline logs foo -n bar
+  # show logs interactively for no inputs
```

Comment:
this needs to rephrased, how about:  "interactive mode: shows log of the selected pipeline run"